threedi_modelchecker/checks/raster.py

```diff
@@ -210,6 +209,35 @@ def description(self):
         return f"The raster in {self.column_name} has non-square raster cells."
 
 
+class RasterGridSizeCheck(BaseRasterCheck):
+    """Check whether the global settings' grid size is an even multiple of a raster's cell size (at least 2x).
+
+    Only works locally.
```

Comment:
Can you get rid of this remark? It in fact works on the server as well. Also goes for the other ones.